lib/ssh_scan/policy.rb

```diff
@@ -25,5 +26,20 @@ def self.from_string(string)
       opts = YAML.load(string)
       self.new(opts)
     end
+
+    # Choose a policy file based on rules
+    def self.first_match(result)
+      Dir.glob("policies/**/*.yml") do |policy_file|
+        next if policy_file == @fallback_file
+        file = YAML.load_file(policy_file)
+        next unless file["rules"]
+        puts "Matching: #{policy_file}"
+        if SSHScan::RuleEngine.match(file["rules"], result)
+          puts "Matched: #{policy_file}"
+          policy_file
```

Comment:
I can do that to make it clearer, but essentially return ... will do the same thing, right?